### ISSUE TITLE
Error parsing JSON: unknown keyword "None"

### ISSUE THREAD

**Issue by ilkkapeltola** [transferwise/pipelinewise-target-snowflake#244]
**Describe the bug**
I'm trying to load data into snowflake from stripe using (https://github.com/singer-io/tap-stripe) and some data objects fail to load with an error 
Error parsing JSON: unknown keyword "None"

The records look like this:

`1860,"[{\"id\": \"txn_xxxxxxxxxxxxxxxxxxxxxxxxxxxx\"}]","py_xxxxxxxxxxxxxxxxxxxxxxxxxxxx","2019-11-27T22:47:20.000000Z","eur","{'access_activity_log': None, 'billing_address': None, 'cancellation_policy': None, 'cancellation_policy_disclosure': None, 'cancellation_rebuttal': None, 'customer_communication': None, 'customer_email_address': None, 'customer_name': None, 'customer_purchase_ip': None, 'customer_signature': None, 'duplicate_charge_documentation': None, 'duplicate_charge_explanation': None, 'duplicate_charge_id': None, 'product_description': None, 'receipt': None, 'refund_policy': None, 'refund_policy_disclosure': None, 'refund_refusal_explanation': None, 'service_date': None, 'service_documentation': None, 'shipping_address': None, 'shipping_carrier': None, 'shipping_date': None, 'shipping_documentation': None, 'shipping_tracking_number': None, 'uncategorized_file': None, 'uncategorized_text': None}","{\"due_by\": \"1970-01-01T00:00:00.000000Z\", \"has_evidence\": false, \"past_due\": false, \"submission_count\": 0}","pdp_xxxxxxxxxxxxxxxxxxxxxxxxxxxx",false,true,"{}","dispute","general","lost","2019-11-27T22:47:20.000000Z"`

I have set the validate_records to true but it doesn't catch any malformatted JSON.
However, when the data is being loaded into snowflake, the load fails with an error.

**To Reproduce**
1. install and configure tap-stripe and pipelinewise-target-snowflake
2. Run the command elt tap-stripe target-sf-transferwise
3. See error Error parsing JSON: unknown keyword "None"

**Expected behavior**
The ELT should complete without errors
OR
the JSON validation should catch malformatted JSON

**Your environment**
 - Version of target: 1.14.1
 - Version of python 3.6

** additional details **

my file format configuration:
`CREATE OR REPLACE FILE FORMAT meltano.prod.default_file_format
TYPE = 'CSV'
ESCAPE='\\'
FIELD_OPTIONALLY_ENCLOSED_BY='"'
ESCAPE_UNENCLOSED_FIELD= '\\';
`

I have tried with table staging and through s3, same issue.
I have stored the staged file in S3 that causes the issue, if that helps. I can share it privately if it helps.

**Comment by Samira-El** [comment]
Hello, can you share the target logs and full error traceback?

**Comment by ilkkapeltola** [comment]
[error-tracelog.txt](https://github.com/transferwise/pipelinewise-target-snowflake/files/7860970/error-tracelog.txt)
[elt.log](https://github.com/transferwise/pipelinewise-target-snowflake/files/7860979/elt.log)

here you go! The logs are run with log-level=debug so they're quite verbose

**Comment by Samira-El** [comment]
The target logs don't really say much. In the record you shared, there are string representation of Python dictionaries, rather than proper json, like this one which I believe is the `evidence` property: 
```
"{'access_activity_log': None, 'billing_address': None, 'cancellation_policy': None, 'cancellation_policy_disclosure': None, 'cancellation_rebuttal': None, 'customer_communication': None, 'customer_email_address': None, 'customer_name': None, 'customer_purchase_ip': None, 'customer_signature': None, 'duplicate_charge_documentation': None, 'duplicate_charge_explanation': None, 'duplicate_charge_id': None, 'product_description': None, 'receipt': None, 'refund_policy': None, 'refund_policy_disclosure': None, 'refund_refusal_explanation': None, 'service_date': None, 'service_documentation': None, 'shipping_address': None, 'shipping_carrier': None, 'shipping_date': None, 'shipping_documentation': None, 'shipping_tracking_number': None, 'uncategorized_file': None, 'uncategorized_text': None}"
```

I reckon the tap is not properly generating the singer records by not using `json.dump`. I don't have a stripe account so can't reproduce. If you can instead share a subset of singer records generated by the tap that trigger this issue, I can investigate further.

**Comment by ilkkapeltola** [comment]
I'd love to! How do I do that? 😁

**Comment by Samira-El** [comment]
You'll have to get your hands a bit dirty and skip meltano, you need to run the tap on its own like outlined here: https://github.com/singer-io/tap-stripe#usage

You can redirect the singer messages to a file, then feed it to target-snowflake with `cat file > target-snowflake -c config.json`

You can extract `disputes` data only since that's where the offending record is.

**Comment by ilkkapeltola** [comment]
Sorry for late reply. Okay thanks, I'll investigate!

**Comment by ilkkapeltola** [comment]
You are correct. Evidence isn't JSON, but stringified, double-quote enclosed 

`"evidence": "{'access_activity_log': None, 'billing_address': None, 'cancellation_policy': None, 'cancellation_policy_disclosure': None, 'cancellation_rebuttal': None, 'customer_communication': None, 'customer_email_address': None, 'customer_name': None, 'customer_purchase_ip': None, 'customer_signature': None, 'duplicate_charge_documentation': None, 'duplicate_charge_explanation': None, 'duplicate_charge_id': None, 'product_description': None, 'receipt': None, 'refund_policy': None, 'refund_policy_disclosure': None, 'refund_refusal_explanation': None, 'service_date': None, 'service_documentation': None, 'shipping_address': None, 'shipping_carrier': None, 'shipping_date': None, 'shipping_documentation': None, 'shipping_tracking_number': None, 'uncategorized_file': None, 'uncategorized_text': None}"`

Attached are the first 4 rows of the tap output file.
[results.txt](https://github.com/transferwise/pipelinewise-target-snowflake/files/8077736/results.txt)
